main.go

```diff
@@ -72,19 +75,23 @@ func main() {
 		os.Exit(1)
 	}
 
-	if err = (&controllers.RouteMonitorReconciler{
+	routeMonitorReconciler := &routemonitor.RouteMonitorReconciler{
 		Client: mgr.GetClient(),
 		Log:    ctrl.Log.WithName("controllers").WithName("RouteMonitor"),
 		Scheme: mgr.GetScheme(),
-	}).SetupWithManager(mgr); err != nil {
+	}
+	if err = routeMonitorReconciler.SetupWithManager(mgr); err != nil {
 		setupLog.Error(err, "unable to create controller", "controller", "RouteMonitor")
 		os.Exit(1)
 	}
+
 	// +kubebuilder:scaffold:builder
 
 	setupLog.Info("starting manager")
 	if err := mgr.Start(ctrl.SetupSignalHandler()); err != nil {
 		setupLog.Error(err, "problem running manager")
 		os.Exit(1)
 	}
+
+	setupLog.V(1).Info("No man's land, don't put things here")
```

Comment:
what's this one about?

Reply:
when I tried to add the `CreateBlackBoxExporter` there, it didn't run, as `mgr.Start` is blocking

Reply:
maybe we make the error message a tiny bit more meaningful to reflect that ;)

Reply:
it's a log message, will update with this fix. thanks!

Reply:
rewrote, can you review the new message and see if it's better?